.github/workflows/pr-type-category.yml

```diff
@@ -24,9 +26,30 @@ jobs:
           ) || (
             contains(github.event.pull_request.labels.*.name, 'New Feature') &&
             contains(github.event.pull_request.labels.*.name, 'Bug-Fix')
+          ) || (
```

Comment:
I don't have a good solution for this off the top of my head, but this conditional looks like it is starting to get a little out of hand.

Reply:
Jack had a suggestion that he posted on a Slack thread. I asked if he could take a look at it and see if there's an alternative. I just extended whatever Jack had set up before as a quick fix.